### PR TITLE
Basic Sign Up

### PR DESCRIPTION
**Pull request by Moonforeva**

### CODE REVIEW

**Comment by Moonforeva** [comment]
Ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -92,3 +95,95 @@ footer {
...
+    margin: 0 0 30px 0;
+  }
+}
+
+.field_with_errors {
+  @extend .has-error;
+  .form-control {
+    color: $state-danger-text;
+  }
+}
```
Comment (ghost) [inline app/assets/stylesheets/custom.scss]: end line

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by params[:id]
```
Comment (ghost) [inline app/controllers/users_controller.rb]:
```suggestion
    @user = User.find_by id: params[:id]
```

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <%= f.email_field :email, class: "form-control" %>
+        
+        <%= f.label :password %>
+        <%= f.password_field :password, class: "form-control" %>
+        
+        <%= f.label :password_confirmation, "Confirmation" %>
+        <%= f.password_field :password_confirmation, class: "form-control" %>
+        
+        <%= f.submit "Create my account", class: "btn btn-primary" %>
+        <% end %>
```
Comment (ghost) [inline app/views/users/new.html.erb]:
```suggestion
        <%= form_for @user do |f| %>
          <%= render "shared/error_messages", object: f.object %>

          <%= f.label :name %>
          <%= f.text_field :name, class: "form-control" %>
        
          <%= f.label :email %>
          <%= f.email_field :email, class: "form-control" %>
        
          <%= f.label :password %>
          <%= f.password_field :password, class: "form-control" %>
        
          <%= f.label :password_confirmation, "Confirmation" %>
          <%= f.password_field :password_confirmation, class: "form-control" %>
        
          <%= f.submit "Create my account", class: "btn btn-primary" %>
        <% end %>
```

**Comment by Moonforeva** [comment]
Ready

**Comment by Moonforeva** [comment]
Ready

**Pull request merged**